doc/classes/ResourceImporterScene.xml

```diff
@@ -27,6 +27,9 @@
 		<member name="animation/trimming" type="bool" setter="" getter="" default="false">
 			If [code]true[/code], trim the beginning and end of animations if there are no keyframe changes. This can reduce output file size and memory usage with certain 3D scenes, depending on the contents of their animation tracks.
 		</member>
+		<member name="hints/use_import_hints" type="bool" setter="" getter="" default="true">
+			Enables or disables the use of import hints. Disable when automatic transformations based on naming conventions are not desired. For more information on import hints, see [url=https://docs.godotengine.org/en/stable/tutorials/assets_pipeline/importing_scenes.html#import-hints]the documentation[/url].
```

Comment:
```suggestion
			Enables or disables the use of import hints. Disable when automatic transformations based on naming conventions are not desired. For more information on import hints, see [url=$DOCS_URL/tutorials/assets_pipeline/importing_scenes.html#import-hints]the documentation[/url].
```

Reply:
We usually start descriptions for boolean properties with `If [code]true[/code], ...`.
So this could be:
```suggestion
			If [code]true[/code], enables the use of import hints. Disable when automatic transformations based on naming conventions are not desired. For more information on import hints, see [url=$DOCS_URL/tutorials/assets_pipeline/importing_scenes.html#import-hints]the documentation[/url].
```